This patch adds support for GCC 5.4 to the GAMESS-US build scripts.

Written by Oliver Stueker, ACENET/Compute Canada
Memorial University of Newfoundland, St. John's, Canada
--- config.orig	2017-07-18 19:34:32.056678893 +0000
+++ config	2017-07-18 19:48:37.701737302 +0000
@@ -410,6 +410,7 @@
          case 5.1:
          case 5.2:
          case 5.3:
+         case 5.4:
             echo "   Good, the newest gfortrans can compile REAL*16 data type."
             breaksw
          default:
@@ -810,6 +811,7 @@
    case 5.1:
    case 5.2:
    case 5.3:
+   case 5.4:
       echo "   Good, the newest gfortrans can compile REAL*16 data type."
       breaksw
    default:
--- comp.orig	2017-07-19 11:55:05.490262037 +0000
+++ comp	2017-07-19 11:52:47.986565565 +0000
@@ -1593,6 +1593,7 @@
          case 5.1:
          case 5.2:
          case 5.3:
+         case 5.4:
             if ($MODULE == cosmo)  set OPT='-O0'
             if ($MODULE == dcscf)  set OPT='-O0'
             if ($MODULE == tddgrd) set OPT='-O0'
@@ -1823,6 +1824,7 @@
          case 5.1:
          case 5.2:
          case 5.3:
+         case 5.4:
             if ($MODULE == cosmo)  set OPT='-O0'  # same issue as seen in 4.6
             if ($MODULE == dcscf)  set OPT='-O0'  # exam44, continues from 4.7
             if ($MODULE == tddgrd) set OPT='-O0'  # exam41, continues from 4.6
@@ -2074,6 +2076,7 @@
          case 5.1:
          case 5.2:
          case 5.3:
+         case 5.4:
             if ($MODULE == cosmo)  set OPT='-O0'
             if ($MODULE == dcscf)  set OPT='-O0'
             if ($MODULE == tddgrd) set OPT='-O0'
@@ -2191,6 +2194,7 @@
          case 5.1:
          case 5.2:
          case 5.3:
+         case 5.4:
             if ($MODULE == cosmo)  set OPT='-O0'
             if ($MODULE == dcscf)  set OPT='-O0'
             if ($MODULE == tddgrd) set OPT='-O0'
